hazelcast/src/test/java/com/hazelcast/map/impl/MapStatisticsAwareServiceTest.java

```diff
@@ -39,11 +39,11 @@
 public class MapStatisticsAwareServiceTest extends HazelcastTestSupport {
 
     @Test
-    public void getStats() {
+    public void getStats_whenMapHasData() {
         HazelcastInstance hz = createHazelcastInstance();
         warmUpPartitions(hz);
         // when one map exists
-        hz.getMap("map");
+        hz.getMap("map").put(1, 1);
         MapService mapService = getNodeEngineImpl(hz).getService(MapService.SERVICE_NAME);
         Map<String, LocalMapStats> mapStats = mapService.getStats();
 
```

Comment:
I just realized the proxy is already created on the joining side by the post join operation, so please disregard the comment above.